apps/ui/src/utils/__tests__/is-span-error.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { ListTraceGroupsResponse } from "@graphql-debugger/types";
+
+import { faker } from "@faker-js/faker";
+
+import { isSpanError } from "../is-trace-error";
+
+const span: ListTraceGroupsResponse["traces"][0]["spans"][0] = {
```

Comment:
```suggestion
import { Span } from "@graphql-debugger/types";

import { faker } from "@faker-js/faker";

import { isSpanError } from "../is-trace-error";

const span: Span = {
```

We might be able to use the Span type here.